.github/workflows/verify-deleted-file-entries.yml

```diff
@@ -0,0 +1,38 @@
+name: Verify Deleted Slide Entries
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  verify-deleted-slides:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: |
+          # Install your dependencies here.
+
+      - name: List deleted slides
+        id: deleted-slides
+        run: |
+          deleted_slides=$(git diff --name-only HEAD^ HEAD)
```

Comment:
I think this will include all _modified_ slides, right?

Also, this only compares HEAD^ to HEAD -- what if the PR has multiple commits in it?

Reply:
updated it in my new commit .